circle.yml

```diff
@@ -0,0 +1,17 @@
+dependencies:
+  override:
+    - pip install docker-compose 
+    - pip install requests
+
+machine:
+    environment:
+        SECRET_KEY: CI
+    services:
+        - docker
+
+test:
+  pre:
+    - docker-compose -f dockereve-master/docker-compose.yml build
+  override:
+    - docker-compose -f dockereve-master/docker-compose.yml up && python test/testGetPost.py
```

Comment:
You are running docker-compose in an interactive way - it will never exit and python test/testGetPost.py. You need to run it as a daemon (`-d`) wait until it boots up and then run the tests.